web/themes/custom/genlyd_aarhus/source/_patterns/02-components/add-activity.twig

```diff
@@ -0,0 +1,626 @@
+                    {% include "@elements/forms/text-field.twig" with { type: "text", "name": "field_time_start" } %}
+                  {% endif %}
+
+                {% endblock %}
+              {% endembed %}
+              {% embed "@modules/grid/grid-item.twig" %}
+                {% block content %}
+
+                  {% if form %}
+                    {# Drupal #}
+                    {{ form.field_time_end }}
+                  {% else %}
+                    {# Pattern Lab #}
+                    {% include "@elements/forms/form-label.twig" with { formLabel: { label: "Sluttidspunkt" } } %}
+                    {% include "@elements/forms/text-field.twig" with { type: "text", "name": "field_time_end" } %}
+                  {% endif %}
+
+                {% endblock %}
+              {% endembed %}
+              {% embed "@modules/grid/grid-item.twig" %}
+                {% block content %}
+
+                  {% if form %}
+                    {# Drupal #}
+                    {{ form.field_zipcode }}
+                  {% else %}
+                    {# Pattern Lab #}
+                    {% include "@elements/forms/form-label.twig" with { formLabel: { label: "Postnummer" } } %}
+                    {% include "@elements/forms/text-field.twig" with { type: "text", "name": "field_zipcode" } %}
+                  {% endif %}
+
+                {% endblock %}
+              {% endembed %}
+              {% embed "@modules/grid/grid-item.twig" %}
+                {% block content %}
+
+                  {% if form %}
+                    {# Drupal #}
+                    {{ form.field_area }}
+                  {% else %}
+                    {# Pattern Lab #}
+                    {% include "@elements/forms/form-label.twig" with { formLabel: { label: "Område" } } %}
+                    {% include "@elements/forms/text-field.twig" with { type: "text", "name": "field_area" } %}
+                  {% endif %}
+
+                {% endblock %}
+              {% endembed %}
+              {% embed "@modules/grid/grid-item.twig" %}
+                {% block content %}
+
+                  {% if form %}
+                    {# Drupal #}
+                    {{ form.field_address }}
+                  {% else %}
+                    {# Pattern Lab #}
+                    {% include "@elements/forms/form-label.twig" with { formLabel: { label: "Adresse" } } %}
+                    {% include "@elements/forms/text-field.twig" with { type: "text", "name": "field_address" } %}
+                  {% endif %}
+
+                {% endblock %}
+              {% endembed %}
+
+            {% endblock %}
+          {% endembed %}
+
+        {% endblock %}
+      {% endembed %}
+
+      {% embed "@modules/grid/grid-item.twig" %}
+        {% block content %}
+
+          {{ form.actions.submit }}
+          {{ form.actions.publish }}
+
+        {% endblock %}
+      {% endembed %}
+
```

Comment:
Waiting for merge with 5 step form code, after merge I will take a look at getting it to work in Pattern Lab.